tests/integration/test_remoted/test_socket_communication/test_ping_pong_message.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.monitoring import FileMonitor, make_callback, REMOTED_DETECTOR_PREFIX
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools.agent_simulator import send_ping_pong_messages
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_socket_communication.yaml')
+
+parameters = [
+    {'PROTOCOL': 'UDP', 'CONNECTION': 'secure', 'PORT': 1514},

```

Comment:
In this case `secure` is constant, so you can remove it from this list and use it as a constant in the template of the test configuration